tests/test_browser.py

```diff
@@ -3793,30 +3809,35 @@ def test_pthread_gcc_atomic_fetch_and_op(self):
         self.btest_exit(test_file('pthread/test_pthread_gcc_atomic_fetch_and_op.cpp'), args=args + ['-s', 'INITIAL_MEMORY=64MB', '-s', 'USE_PTHREADS', '-s', 'PTHREAD_POOL_SIZE=8'])
 
   # 64 bit version of the above test.
+  @also_with_wasm2js
```

Comment:
Does this work with btest? We normally use put emcc args in a list there, like in the btest invocation here. Do we also pick up `self.set_setting` and `self.emcc_args` changes (like the decorator does)?

Reply:
Yes, best honors those things these days.  I added `self.get_emcc_args()` to the command line.